src/modules/endpoint/sagas/loadLanguages.js

```diff
@@ -6,14 +6,17 @@ import { createLanguagesEndpoint, LanguageModel, Payload } from '@integreat-app/
 import request from '../request'
 import { baseUrl } from '../constants'
 import type { DataContainer } from '../DataContainer'
+import DatabaseContext from '../DatabaseContext'
 
-export default function * loadLanguages (city: string, dataContainer: DataContainer, shouldUpdate: boolean): Saga<void> {
-  if (dataContainer.languagesAvailable() && !shouldUpdate) {
+export default function * loadLanguages (context: DatabaseContext, dataContainer: DataContainer, shouldUpdate: boolean): Saga<void> {
+  const languagesAvailable = yield call({context: dataContainer, fn: dataContainer.languagesAvailable}, context)
```

Comment:
```suggestion
  const languagesAvailable = yield call(dataContainer.languagesAvailable, context)
```

Reply:
Would not work that way because languagesAvailable references this.